src/client/src/models/Scene.js

```diff
@@ -69,6 +75,14 @@ const Scene = {
   setCurrent: function(sceneId) {
     const list = Scene.getScenes()
     Scene.current = list[sceneId]
+  },
+
+  setInputFormat: function(format) {
+    const list = JSON.parse(localStorage.getItem('input-format'))
+    list['inputFormat'] = format
```

Comment:
I Forgot to add a comment!! You have it as list['inputFormat'] it needs to be list['input-format']